infrastructure/cli/cmd/config/handle_configmaps.go

```diff
@@ -0,0 +1,48 @@
+package config
+
+import (
+	"context"
+
+	corev1 "k8s.io/api/core/v1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+//UpdateSourceCM Updates the config map of a source given a new map
+func UpdateSourceCM(source string, newCmData map[string]string, kubeConfigFile string) error {
+	config, err := clientcmd.BuildConfigFromFlags("", kubeConfigFile)
+	configMapName := "sources-" + source
+	if err != nil {
+		return err
+	}
+
+	// creates the clientset
+	clientset, err := kubernetes.NewForConfig(config)
+	if err != nil {
+		return err
+	}
+
+	cm, err := clientset.CoreV1().ConfigMaps("default").Get(context.TODO(), configMapName, v1.GetOptions{})
+	if cm.GetName() == "" {
+		_, err = clientset.CoreV1().ConfigMaps("default").Create(context.TODO(),
+			&corev1.ConfigMap{
+				ObjectMeta: v1.ObjectMeta{
+					Name:      configMapName,
+					Namespace: "default",
```

Comment:
this should come from the config? a little tricky maybe

Reply:
I would do that we get it from the `airy.yaml` file. If it is not there - we have a `default` default.

Reply:
yeah that's a good place!